src/utils/filter-utils.js

```diff
@@ -288,9 +389,10 @@ export function isDataMatchFilter(data, filter, i) {
       return isInRange(val, filter.value);
 
     case FILTER_TYPES.timeRange:
-      const timeVal = filter.mappedValue
-        ? filter.mappedValue[i]
-        : moment.utc(val).valueOf();
+      const timeVal = field && field.filterProp && field.filterProp.mappedValue
```

Comment:
`Array.isArray(field.filterProp.mappedValue)` is a safer then `field.filterProp.mappedValue`